maisim/maisim.Game/Beatmaps/DifficultyRating.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace maisim.Game.Beatmaps
+{
+    public enum DifficultyRating
```

Comment:
mega small nit not mandatory: maybe a small xmldoc there

Reply:
added